packages/merge-on-green/src/merge-logic.ts

```diff
@@ -214,6 +229,17 @@ mergeOnGreen.getRequiredChecks = async function getRequiredChecks(
           console.log(
             `Your language's required checks were overridden because of the repo ${owner}/${repo}`
           );
+          if (
+            isOverriden.requiredStatusChecks[0] === 'Native branch protection'
```

Comment:
rather than making this a string, part of me would rather this be an additional parameter on overrides, something like:

```
repoOverrides: [
  {
    repo: 'googleapis/blerg',
    useBranchProtectionRules: true
]
```

Reply:
I like it!!